src/main/java/hudson/plugins/sonar/SonarRunnerBuilder.java

```diff
@@ -66,7 +71,6 @@
 import javax.annotation.Nonnull;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'javax.annotation.Nonnull'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.internal.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)